dijet/config/categories.py

```diff
@@ -2,39 +2,337 @@
 
 """
 Definition of categories.
-"""
 
-from collections import OrderedDict
+Categories are assigned a unique integer ID according to a fixed numbering
+scheme, with digits/groups of digits indicating the different category groups:
+
+               lowest digit
+                          |
+    +---+---+---+---+---+---+
+    | I | A | P | P | E | E |
+    +---+---+---+---+---+---+
+
++=======+===============+======================+=================================+
+| Digit | Description   | Values               | Category name                   |
++=======+===============+======================+=================================+
+| E     | eta bins      | 1 to e               | eta_{eta_min}_{eta_max}         |
+|    10 |               |                      |                                 |
++-------+---------------+----------------------+---------------------------------+
+| P     | pt avg bin    | 1 to p               | pt_{pt_min}_{pt_max}            |
+|  1000 |               |                      |                                 |
++-------+---------------+----------------------+---------------------------------+
+| A     | alpha binning | 1 to a               | alpha_{min}_{max}               |
+| x 1e4 |               |                      |                                 |
++-------+---------------+----------------------+---------------------------------+
+| I     | Inclusive     |                      |                                 |
+|       | alpha         |                      | In combination with alpha bins  |
+|       |               | 0: exclusive         | Excl: alpha in one bin          |
+|       |               | 1: inclusive         | Incl: All alpha below the upper |
+|       |               |                      |       bin value                 |
+|       |               |                      | e.g.  bin IA=06 -> 0.25<a<0.3   |
+| x 1e5 |               |                      |       bin IA=16 ->      a<0.3   |
++-------+---------------+----------------------+---------------------------------+
```

Comment:
Thanks for adding the info about how the `AI` is defined. I would also suggest filling in the category name for the inclusive alpha bins for completeness. And actually, since those two digits both describe an alpha bin together, one could also merge the cells in the "Category name" column, for example like this:
```suggestion
+-------+---------------+----------------------+---------------------------------+
| A     | alpha binning | 1 to a               | alpha_{min}_{max} (exclusive)   |
| x 1e4 |               |                      | or                              |
+-------+---------------+----------------------+ alpha_lt_{max} (inclusive)      |
| I     | Inclusive     |                      |                                 |
|       | alpha flag    |                      | In combination with alpha bins  |
|       |               | 0: exclusive         | Excl: alpha in one bin          |
|       |               | 1: inclusive         | Incl: All alpha below the upper |
|       |               |                      |       bin value                 |
|       |               |                      | e.g.  bin IA=06 -> 0.25<a<0.3   |
| x 1e5 |               |                      |       bin IA=16 ->      a<0.3   |
+-------+---------------+----------------------+---------------------------------+
```